Uplift/Views/GymDetail/Facilities/PriceInformationCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  PriceInformationCell.swift
+//  Uplift
+//
+//  Created by Yana Sang on 10/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class PriceInformationCell: UICollectionViewCell {
+
+    struct Constants {
+        static let lineHeight: CGFloat = 20
+        static let padding: CGFloat = 12
+    }
+
+    // MARK: - Private view vars
+    private let background = UIView()
+    private let itemsList = UITextView()
+    private let pricesList = UITextView()
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func configure(items: [String], prices: [String]) {
+        itemsList.attributedText = generateAttributedString(for: items, font: ._14MontserratRegular)
+        pricesList.attributedText = generateAttributedString(for: prices, font: ._14MontserratSemiBold, alignment: .right)
+
+        pricesList.snp.remakeConstraints { make in
+            make.top.equalTo(background).offset(Constants.padding)
+            make.trailing.bottom.equalTo(background).inset(Constants.padding)
+            make.width.equalTo(pricesList.intrinsicContentSize.width)
+        }
+    }
+
+    private func setupViews() {
+        background.layer.cornerRadius = 4.0
+        background.backgroundColor = .gray01
+        contentView.addSubview(background)
+
+        itemsList.backgroundColor = .gray01
+        itemsList.isEditable = false
+        itemsList.isSelectable = false
+        itemsList.isScrollEnabled = false
+        itemsList.textContainerInset = .zero
+        itemsList.textContainer.lineFragmentPadding = 0
+        contentView.addSubview(itemsList)
```

Comment:
Y r we not adding to background instead? Seems like the constraints r all relative to background anyway